### ISSUE TITLE
Make backups "download" to the PV instead of the browser

### ISSUE THREAD

**Issue by tnelson-doghouse** [uselagoon/lagoon#2959]
As a user, I want to avoid having to download/upload backups.  

When a backup has been Retrieved, and is available for Download, it would be nice if it could be "downloaded" into the PV instead of to the user's machine.  I'm thinking with a tgz file in a /restores folder or something.  

Advantages:

- This would save the download/upload process (which is a problem from a slower connection, and could be significantly important when doing an emergency restore).
- This may (or may not) be one step in the chain to achieving https://github.com/uselagoon/lagoon/issues/2888
- This would work with any site type (Drupal or non-Drupal), as long as we could find the PV

**Comment by shreddedbacon** [comment]
Just a few concerns around this.
Not all backups will have a PV that they can be restored to properly (MySQL/MariaDB for example if they are using the shared DBaaS).

Restoring some things to a PV could also have the potential to be served by a web server, depending on the configuration and type of service that the backup is for.

**Comment by tnelson-doghouse** [comment]
Also cool (ie. instead of my other suggestion) would be if we could mount an additional PV (readonly) that already has the backup tgz files in it (maybe direct from S3, either a new temporary bucket or the regular one, depending on security implications).  This would mean that:
- Because of the mount point being in a different place (maybe lagoon-restores), the files wouldn't be servable to the public
- Some containers might not have a PV, but if this were done, they could still use it.

**Comment by tobybellwood** [comment]
This could actually be a really nice example of an advanced task that could use the lagoon-cli to get the S3 link and then copy it to a destination folder in a service.

**Comment by tnelson-doghouse** [comment]
Would this also apply to https://github.com/uselagoon/lagoon/issues/2888 ?

**Comment by shreddedbacon** [comment]
> Would this also apply to #2888 ?

Yes, you could probably use tasks to do something similar for that
